AdminDashboard/AdminDashboard.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>$(NoWarn);1591</NoWarn>
```

Comment:
What warning are we disabling here (and why?)

Reply:
Warning ignored: "Missing XML comment for publicly visible type or member"

This is to enable documentation on the web APIs with Swagger!

Reply:
Thanks for the info